aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,58 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## April 4, 2022 - April 8, 2022
+
+### Latest Releases
+
+- **grommet-theme-hpe v3.0.0** - This version ships with additional responsive breakpoints to help deliver the optimal user experience for HPE customers.
+  - Gain precise control over how page layouts respond and adapt.
+  - Implement grid layouts, such as a dashboard of cards, which gracefully adapt and flow.
+  - These additional breakpoints are breaking changes from v2 that will require some minor code changes to ensure backwards compatibility. A [migration guide](https://github.com/grommet/grommet-theme-hpe/wiki/Migrating-from-HPE-Theme-2.x-to-3.x) is available to make this a smooth process.
+
+## March 9, 2022 - April 1, 2022
+
+### Communications
+
+- We are actively working on a process for receiving design contributions from members outside of the Design System core team.
+- We will begin this process by rolling out a way for designers to contribute small fixes and medium enhancements by leveraging the Figma Branching feature. Stay tuned!
+
+
+### Roadmap Updates
+
+- **Page Layouts**
+  - Updated the [Page Layouts Figma file](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=2891%3A21273) to reflect new Page component in Grommet. Page component guidance and examples coming soon to the Design System site.
+- **Global Banner Notifications**
+  - Guidance and examples are now available on the [Design System site](/templates/global-banner-notifications).
+  - Theme-related styling is available on `grommet-theme-hpe v2.4.0`.
+- **Page Header**
+  - Finalized gathering use cases, annotating designs from service teams, and exploring anatomy. [Design ideations](https://www.figma.com/file/BHp0KrOcPvbnzNyshfHkd8/HPE-Page-Header) are in progress.
```

Comment:
I think this is too "us" focussed as opposed to focus on our reader. We should always be asking "What is important to the reader? Why should they care?"

In this case, why they should care is 1) awareness that work is actively being done to standardize page headers across HPE applications which will likely impact them, 2) where we are in the process so that if they have thoughts to share or contribute they know when and how they can engage and provide input or point of view on the topic, as well as how soon they may be affected.

Also, this link points to a blank "finished" page. The intent behind the link is helpful.

```suggestion
  - Began work to standardize Page Headers across HPE applications. The scope of this work is to define how page titles, page level actions, and contextual navigation should be displayed.
  - Completed research and exploration phase by collecting use cases across HPE applications and identified commonalities and deviances.
  - [Design ideations](https://www.figma.com/file/BHp0KrOcPvbnzNyshfHkd8/HPE-Page-Header) are currently being explored.
```

Fix link. Consider some sort of invitation to the reader to provide feedback.